core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -589,10 +589,11 @@ private[spark] class BlockManager(
   private def doGetRemote(blockId: BlockId, asBlockResult: Boolean): Option[Any] = {
     require(blockId != null, "BlockId is null")
     val locations = Random.shuffle(master.getLocations(blockId))
+    val timeout = conf.getInt("spark.storage.fetchBlockTimeout", 1000) // seconds
```

Comment:
i think it should be quite large than 1000s. but i think we do not need fetchBlockTimeout because fetchBlocks has network.timeout for nio and netty. can you describe what conditions we need fetchBlockTimeout? thanks.